src/vuestic-theme/vuestic-components/va-medium-editor/VaMediumEditor.vue

```diff
@@ -69,4 +69,117 @@ export default {
     outline: none;
   }
 }
+
+.medium-editor-toolbar,
```

Comment:
Would make sense to move this to `.va-medium-editor` block. As user can potentially have other medium editors on his page.